lib/ansible/modules/storage/ibm/ibm_sa_domain.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2018 IBM CORPORATION
+# Author(s): Tzur Eliyahu <tzure@il.ibm.com>
+#
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: ibm_sa_domain
+short_description: Adds domains to or removes them from IBM Spectrum Accelerate storage systems.
+version_added: "2.8"
+
+description:
+    - "This module adds domains to or removes them
+        from IBM Spectrum Accelerate storage systems"
+
+options:
+    domain:
+        description:
+            - domain name
+        required: true
+    state:
+        description:
+            - The desired state of the domain
+        required: true
+        default: "present"
+        choices: [ "present", "absent" ]
+    ldap_id:
+        description:
+            - ldap id to add to the domain.
+        required: false
+    size:
+        description:
+            - domain size
+        required: false
+    hard_capacity:
+        description:
+            - Hard capacity of the domain.
+        required: false
+    soft_capacity:
+        description:
+            - Soft capacity of the domain.
+        required: false
+    max_cgs:
+        description:
+            - Number of max cgs.
+        required: false
+    max_dms:
+        description:
+            - number of max dms.
+        required: false
+    max_mirrors:
+        description:
+            - number of max_mirrors.
+        required: false
+    max_pools:
+        description:
+            - number of max_pools.
+        required: false
+    max_volumes:
+        description:
+            - number of max_volumes.
+        required: false
+    perf_class:
+        description:
+            - Add the domain to a performance class.
+        required: false
+
+extends_documentation_fragment:
+    - ibm_storage
+
+author:
+    - Tzur Eliyahu (tzure@il.ibm.com)
```

Comment:
Add github handle here.
```suggestion
    - Tzur Eliyahu (@tzurE)
```